Assets/_Scripts/Enemy/Enemy.cs

```diff
@@ -58,6 +65,10 @@ public void Shoot()
         currentProjectile.speed = projectileSpeed;
```

Comment:
nit: If you want to use bulletSpeed in the projectile instead of speed this needs to be changed here, this will allow each enemy to set its own speed of the projectile. I know they probably all fall at the same speed but having each enemy have the ability to set their own projectile speed will make it more customizable if we so choose.

Reply:
didn't make any changes to this as the two "Heavy Enemys" they have differing values 
![Screenshot_764](https://github.com/SolomanDavis/VGP145-S24-capstone-retro/assets/120427266/75c0315d-e84a-406f-acf2-b3856a0660ec)
![Screenshot_763](https://github.com/SolomanDavis/VGP145-S24-capstone-retro/assets/120427266/54b072d9-5688-4311-a309-a7827add40ee)

Reply:
In which case I think this needs to be set to currentProjectile.bulletSpeed = projectile Speed instead as this is the variable with in the projectile  actually being used to set its movement. At the minute this projectile speed is just setting a variable that isn't in use.

Reply:
Error CS0122 'EnemyProjectile.bulletSpeed' is inaccessible due to its protection level is the error I am receiving when trying to adopt the method you are asking me to put in

Reply:
I made the [SerializeField] private float bulletSpeed = 10f; in the EnemyProjectile script a public as it seemed to be the way to make it work